api/api/src/main/scala/uk/ac/wellcome/platform/api/models/DisplayAggregation.scala

```diff
@@ -13,6 +13,18 @@ case class DisplayAggregation[T](
   @JsonKey("type") @Schema(name = "type") ontologyType: String = "Aggregation"
 )
 
+case object DisplayAggregation {
+  def apply[T, D](agg: Aggregation[T], display: T => D): DisplayAggregation[D] =
```

Comment:
nit: would suggest a naming change from `D` to `DisplayT` following the convention of the class names themselves